### PR TITLE
Start new internal/iface package

### PR DESCRIPTION
**Pull request by miekg**
Allows to mimimize the similarties between ARP and NDP. Also put
dropReasons and there and made them public.

### CODE REVIEW

**Comment by danderson** [comment]
CircleCI build fails, looks like you forgot to commit the internal/iface package?

**Comment by miekg** [comment]
of course I forgot that ...

**Comment by miekg** [comment]
... and also forgot to gut ndp for this function.

File: internal/iface/iface.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	"fmt"
+	"net"
+
+	"github.com/golang/glog"
+)
+
+// DropReason is the reason why a packet was dropped.
+type DropReason int
+
+const (
```
Comment (danderson) [inline internal/iface/iface.go]: Constant block needs a docstring (or each constant separately, but the entire block is probably enough)

**Comment by danderson** [comment]
Almost there... Build and test works, now only lint checks fail.

**Comment by miekg** [comment]
linted and squashed

**Comment by danderson** [comment]
golint is still sad about that constant block :(

BTW netlify deploy failure is my fault, I'm refactoring the website and there's dead links that make hugo sad. I'll fix that separately.

Approving the change because I have to run out, please make CircleCI happy before merging :)

**Comment by danderson** [comment]
Ah, and it's happy now! I was looking at the previous build.

**Pull request merged**